src/mango/src/mango_selector.erl

```diff
@@ -667,6 +673,53 @@ is_constant_field([{[{_UnMatched, _}]} | Rest], Field) ->
 -ifdef(TEST).
 -include_lib("eunit/include/eunit.hrl").
 
+exists_false_basic_test() ->
+    RequiredFields = [{<<"A">>, false}],
+    Selector = normalize({[{<<"A">>,{[{<<"$exists">>,false}]}}]}),
```

Comment:
just going to be a bit nitpicky here and say be consistent with space behind the comma. this goes for the rest of the testcases and the normalized selectors